umbraco-deploy/troubleshooting.md

```diff
@@ -10,3 +10,67 @@ If you are unable to find the issue you are having, then please reach out to our
 ## Schema mismatches
 
 When transferring or restoring content between two Umbraco Deploy environments, you might run into **Schema mismatch** errors. For more information on how to resolve schema mismatch issues, see the [Schema Mismatches](../umbraco-cloud/troubleshooting/deployments/schema-mismatches.md) article.
+
+Umbraco Deploy maintains a cached set of signatures that represent each schema and content item. They are used when transferring or restoring content between environments to aid performance.
+
+If having resolved schema mismatches you still have reports of errors, it might be that the signatures are out of date.  In other words, Deploy is using a cached representation of an item that no longer matches the actual item stored in Umbraco.
```

Comment:
```suggestion
If having resolved schema mismatches you still have reports of errors, it might be that the signatures are out of date. In other words, Deploy is using a cached representation of an item that no longer matches the actual item stored in Umbraco.
```